internal/zinc-core/src/test/scala/sbt/inc/TestAnalysisCallback.scala

```diff
@@ -58,40 +58,7 @@ class TestAnalysisCallback(
   def hashFile(f: File): Array[Byte] = Stamp.hash(f).asInstanceOf[Hash].value
 
   def get: TestAnalysis = {
-
```

Comment:
Tests for that utils are commented out: https://github.com/sbt/zinc/blob/1.0/internal/zinc-core/src/test/scala/sbt/inc/IncrementalTest.scala